src/js/modules/scrollTopArrow.js

```diff
@@ -0,0 +1,27 @@
+import { throttle } from "throttle-debounce";
+
+const SCROLL_DELAY = 250;
+const PAGE_TOP = 0;
+let button = document.getElementById("buttonScrollTop");
+let viewportHeight = document.documentElement.clientHeight;
+
+button.addEventListener("click", ()=> window.scrollTo({
+	top: PAGE_TOP,
+	behavior: "smooth"
+}));
+
+const throttledMoveToTop = throttle(SCROLL_DELAY, moveToTop);
+
+function moveToTop() {
+	const scrollHeight = window.pageYOffset;
```

Comment:
pageYOffset is outdate property, use actual one

Reply:
Done